.github/workflows/upload-tutorials-stats.yml

```diff
@@ -0,0 +1,41 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Upload tutorials stat
+
+# Controls when the workflow will run
+on:
+  pull_request: # DO NOT MERGE
+  schedule:
+  # Run this once per day.
+    - cron: "0 0 * * *"
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  get-tutorials-stats:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the tutorials repo
+        uses: actions/checkout@v3
+        with:
+          repository: 'pytorch/tutorials'
+          path: './tutorials'
+          fetch-depth: 0
+      - name: Checkout the PyTorch repo
+        uses: malfet/checkout@silent-checkout
+        with:
+          ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+          submodules: recursive
+          path: pytorch
+      - name: Install dependancies
+        run: |
+          python3 -m pip install boto3==1.26.69 typing==3.7.4.3
+
+      - name: Run the script
+        env:
+          AWS_DEFAULT_REGION: us-east-1
+          AWS_ACCESS_KEY_ID: ${{ secrets.aws-access-key-id }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.aws-pytorch-uploader-secret-access-key }}
```

Comment:
Those keys, as name suggest are limited to S3 buckets, they should not be used to create DB connection.

Reply:
I believe the secret is called either secrets.AWS_ACCESS_KEY_ID or secrets.OUR_AWS_ACCESS_KEY_ID, and secrets.AWS_SECRET_ACCESS_KEY or secrets.OUR_AWS_SECRET_ACCESS_KEY, not sure what the difference is and also not sure why we have two...